pkg/resources/pciNetDevice.go

```diff
@@ -82,7 +91,7 @@ func NewPciNetDevice(pciDevice *ghw.PCIDevice, rFactory types.ResourceFactory) (
 		deviceSpecs: dSpecs,
 		mounts:      mnt,
 		env:         env,
-		numa:        "", // TO-DO: Get this using utils pkg
+		numa:        strconv.Itoa(nodenum),
```

Comment:
I know, that this field is never actually used, but shouldn't this be `""` for a `-1` case?

Reply:
Sure, I can do that.  There's some merit to keeping the current behaviour in the no-NUMA case.